src/main/java/com/urswolfer/gerrit/client/rest/http/changes/ChangeApiRestClient.java

```diff
@@ -345,4 +357,8 @@ public void index() throws RestApiException {
     protected String getRequestPath() {
         return "/changes/" + id;
     }
+
+    private String getVersion() throws RestApiException {
+        return serverRestClient.getVersion();
```

Comment:
This needs to be memoized, so that we don't ask Gerrit version all the times.

Reply:
Yes, please do that.

Reply:
Done.